examples/step-65/doc/intro.dox

```diff
@@ -317,7 +372,7 @@ curved mesh elements is the same as the degree of the polynomials for the
 numerical solution. If the degree of the geometry is higher or lower than the
 solution, one calls that a super- or sub-parametric geometry representation,
 respectively. In deal.II, the standard class for polynomial representation is
-MappingQGeneric. If this class is used with polynomial degree $4$ in 3D, a
+MappingQGeneric. If, for example, this class is used with polynomial degree $4$ in 3D, a
 total of 125 ($=(4+1)^3$) points are needed for the tri-cubic
```

Comment:
```suggestion
total of 125 ($=(4+1)^3$) points are needed for the
```
(I know this is a mistake entirely in my original text, but since you are so close it would be great if you could remove it. I was confused by degree 3 that is the degree of the polynomials in the actual code, but the mapping uses one degree higher.)